### ISSUE TITLE
Wired USB-C headphone media buttons/controls always wake screen

### ISSUE THREAD

**Issue by sidshattuck** [GrapheneOS/os-issue-tracker#2927]
Suspect this is an upstream issue since it's only occurred since GrapheneOS moved to Android 14, however I haven't seen more that one or two posts about this anywhere else on the (most likely because this bug does not affect Bluetooth headphones) 

This is specific to wired USB-C headphones of all kinds.
I have tested using Apple USB-C earbuds and a number of other generic brands.

The bug:
When pressing the physical media buttons (volume up/down or pause/play) on the headphone cable, the screen will wake. This is very frustrating when you have the phone in your pocket and waking the screen results in software buttons getting pressed at random.

Here are the only mentions of the bug I've found elsewhere:

https://www.resetera.com/threads/does-anyone-use-wired-headphones-earbuds-on-android-14.781958/
https://www.reddit.com/r/GooglePixel/comments/14i3cuh/my_screen_turns_on_when_i_press_my_headphones/
https://support.google.com/pixelphone/thread/240885988/wired-earbuds-control-button-always-wakes-screen-in-android-14?hl=en

I don't have a Google account so don't know where else to report. Any guidance appreciated.

**Comment by sidshattuck** [comment]
Forgot to mention I've reproduced this bug on Pixel 6a and Pixel 4a

**Comment by min7-i** [comment]
@sidshattuck After updating to 2024061400 the issue doesn't occur anymore on my phone. I'm assuming that this was fixed with Android 14 QPR3.

**Comment by sidshattuck** [comment]
Yes so happy it's been fixed after almost a year... Thanks :smile:
